db/migrate/20231109024110_initial_migration.rb

```diff
@@ -0,0 +1,15 @@
+class InitialMigration < ActiveRecord::Migration[7.1]
+  def change
+    create_table :quotes do |t|
+      t.text :text, null: false, index: { unique: true }
```

Comment:
There's not many Ron Swanson quotes (I checked via https://ron-swanson-quotes.herokuapp.com/v2/quotes/1000), so this will scale.